Code/Source/ROS2ModuleInterface.h

```diff
@@ -8,6 +8,7 @@
 #include <AzCore/Memory/SystemAllocator.h>
 #include <AzCore/Module/Module.h>
 #include <Frame/ROS2FrameComponent.h>
+#include <RobotControl/ROS2RobotControlComponent.h">
```

Comment:
There is an additional `"` character in the include path that shouldn't be here

Reply:
My bad - a merge conflict between <> and "" includes was resolved with this rather ugly child!